src/mn_menu.c

```diff
@@ -3125,8 +3132,11 @@ void M_Drawer(void)
         {
             if (alttext)
             {
-                MN_DrawStringCR(x, y + 8 - MN_StringHeight(alttext) / 2, cr,
-                                NULL, alttext);
+                if (!MN_DrawFon2String(x, y, cr, alttext))
```

Comment:
So, if there are patches then we draw patches, else if there is a big font then we use the big font, else we use the hud font?

Reply:
Yes, correct.